### PR TITLE
Improved Subgraph Docs

### PR DESCRIPTION
**Pull request by PaulieB14**
@kolysapphire.  Made changes and added subgraph info and links to existing subgraph file and also added query examples.

### CODE REVIEW

**Comment by PaulieB14** [comment]
@kolyasapphire I have 2 additional query examples - should I add?

File: docs/integrations/subgraph.md [hunk 1]
```diff
@@ -1,7 +1,143 @@
...
+The schema of GraphQL elements available is defined in [`/schema.graphql` ](https://github.com/lidofinance/lido-subgraph/blob/master/schema.graphql)
+
+Subgraph information is serviced by a decentralized group of server operators called Indexers.
+
+## Ethereum Mainnet
+
+[Creating an API Key Video Tutorial](https://www.youtube.com/watch?v=UrfIpm-Vlgs)
+
+- [Explorer Page](https://thegraph.com/hosted-service/subgraph/lidofinance/lido)
+- Graphql Endpoint: https://gateway.thegraph.com/api/[api-key]/subgraphs/id/HXfMc1jPHfFQoccWd7VMv66km75FoxVHDMvsJj5vG5vf
```
Comment (kolyasapphire) [inline docs/integrations/subgraph.md:23]: GraphQL

File: docs/integrations/subgraph.md [hunk 2]
```diff
@@ -98,45 +98,46 @@ Query description: gather the transfer information
...
+    sender
+    amount
+    block
+    blockTime
+    transactionHash
+  }
+}
+```
+
+## Obtain keys of node operator
```
Comment (kolyasapphire) [inline docs/integrations/subgraph.md:133]: Node Operator Keys

File: docs/integrations/subgraph.md [hunk 3]
```diff
@@ -98,45 +98,46 @@ Query description: gather the transfer information
...
+
+Query description: obtain keys of node operator
```
Comment (kolyasapphire) [inline docs/integrations/subgraph.md:135]: Fetch validator keys of a node operator.

**Comment by kolyasapphire** [comment]
Examples are good now, should be enough, will request a few other changes

**Review by kolyasapphire — CHANGES_REQUESTED** [review]

File: docs/integrations/subgraph.md [hunk 4]
```diff
@@ -1,7 +1,142 @@
...
+- [Code Repo](https://github.com/lidofinance/lido-subgraph/)
+
+## Helpful Links
+
+[Querying from an Application](https://thegraph.com/docs/en/developer/querying-from-your-app/)
+
+[Managing your API Key & Setting your indexer preferences](https://thegraph.com/docs/en/studio/managing-api-keys/)
+
+<br />
+<br />
```
[the remark this answers is not in the document]
Comment (PaulieB14) [reply docs/integrations/subgraph.md:33]: Did this just for spacing.  I can take them out.

**Comment by PaulieB14** [comment]
@kolyasapphire.  I revised and think we are good to go.  If you need any revisions let me know.  Also we are working on automation of getting schema into a markdown file with entities if you are interested using python code.  I know you didn't want entities displayed since so many but maybe if automated it would be easier when updates.

**Comment by kolyasapphire** [comment]
Thanks, but yes, entity data automation not necessary at this time

**Comment by kolyasapphire** [comment]
Last thing - please remove `.vscode/settings.json` and we're ready to go

**Comment by PaulieB14** [comment]
@kolyasapphire I believe it's changed now.   Please let me know

**Comment by kolyasapphire** [comment]
Don't see the change

**Comment by PaulieB14** [comment]
@kolyasapphire  - should be deleted now

**Review by kolyasapphire — APPROVED** [review]

**Comment by kolyasapphire** [comment]
All good now! Will merge and adjust wording a bit. Thanks a lot for your help!

**Pull request merged**